corral/cmd/cmd_run.pony

```diff
@@ -42,6 +44,29 @@ class CmdRun is CmdType
         else
           ctx.env.vars
         end
+      
+      // if we use the exec option, replace this process with the run command
+      // allows for use of things like corral run -- lldb ponyc
+      if exec then
+        ifdef posix then
```

Comment:
This isn't Windows friendly right? Can we do this using the process monitor?

Reply:
Correct.  There is likely a better general solution, I only spent a little time three months ago tweaking it just enough so I could get back to debugging my ponyc.

Corral uses ProcessMonitor already.  I can take another stab at a better solution around that. I'll drop another comment if I find anything, or if I don't and someone else should take look.